tests/parse_pymon.py

```diff
@@ -0,0 +1,61 @@
+"""
+Parse and display test memory.
+
+Uses pytest-monitor plugin from https://github.com/CFMTech/pytest-monitor
+Lots of other metrics can be read from the file via sqlite parsing.,
+currently just MEM_USAGE (RES memory, in MB).
+"""
+import sqlite3
+import sys
+from operator import itemgetter
+
+
+def _parse_pymon_database():
+    # Create a SQL connection to our SQLite database
+    con = sqlite3.connect(".pymon")
+    cur = con.cursor()
+
+    # The result of a "cursor.execute" can be iterated over by row
+    # first look at memory
+    print("Looking for tests that exceed 1GB resident memory.")
+    big_mem_tests = []
+    for row in cur.execute('select ITEM, MEM_USAGE from TEST_METRICS;'):
+        test_name, memory_used = row[0], row[1]
+        if memory_used > 1000.:  # test result in RES mem in MB
+            print("Test name / memory (MB)")
+            print(test_name, memory_used)
+        elif memory_used > 4000.:
+            big_mem_tests.append((test_name, memory_used))
+
+    # then look at total time (in seconds)
+    # (user time is availbale too via USER_TIME, kernel time via KERNEL_TIME)
+    timed_tests = []
+    sq_command = \
+        'select ITEM, ITEM_VARIANT, ITEM_PATH, TOTAL_TIME from TEST_METRICS;'
+    for row in cur.execute(sq_command):
+        test_name, test_var, test_path, time_used = \
+            row[0], row[1], row[2], row[3]
+        timed_tests.append((test_name, test_var, test_path, time_used))
+
+    timed_tests = sorted(timed_tests, reverse=True, key=itemgetter(3))
+    hundred_slowest_tests = timed_tests[0:100]
+    print("List of 100 slowest tests (name, variant: duration [s] and path)")
+    if hundred_slowest_tests:
+        for test_name, test_var, pth, test_duration in hundred_slowest_tests:
+            print(test_name + " " + test_var + ": " + "%.2f" % test_duration +
+                  "s\n" + pth)
```

Comment:
Would it be possible to print the result as
```
34.43s    tests/integration/test_task.py::test_run_tasks[16-spawn]
```
so I can just copy paste the line and run
```
pytest tests/integration/test_task.py::test_run_tasks[16-spawn]
```
to run the slow test locally?
Now it looks like this
```
test_run_tasks test_run_tasks[16, spawn]: 34.43s
tests.integration.test_task
```
and that's still confusing, because the result is spread out over two lines and some things are repeated

Reply:
man, you're harder to please than a French revolutionary in 1789 :rofl: OK, let me plug that in